src/controller/java/CHIPDeviceController-JNI.cpp

```diff
@@ -541,6 +541,17 @@ JNI_METHOD(void, setUseJavaCallbackForNOCRequest)
     AndroidDeviceControllerWrapper * wrapper = AndroidDeviceControllerWrapper::FromJNIHandle(handle);
 
     wrapper->GetAndroidOperationalCredentialsIssuer()->SetUseJavaCallbackForNOCRequest(useCallback);
+
+    if (useCallback)
+    {
+        // if we are assigning a callback, then make the device commissioner delegate verification to the cloud
+        wrapper->Controller()->SetDeviceAttestationVerifier(wrapper->GetCloudDACVerifier());
```

Comment:
We don't need a "CloudDACVerifier" member. You could call it "setCustomDacVerifier" and this is the `GetCustomDACVerifier`. It may not use a cloud at all. No "cloud" anything should show up in Matter SDK common code if possible

Reply:
Renamed this to PartialDACVerifier